File: ha/mods/com.burrsutter~senderjava~1.0/Sender.java
import org.vertx.java.core.Handler;
import org.vertx.java.core.eventbus.Message;
import org.vertx.java.platform.Verticle;

public class Sender extends Verticle {

  public void start() {
    // Publish some news on the feed every second
    vertx.setPeriodic(1000, new Handler<Long>() {
      @Override
      public void handle(Long timerID) {
			  System.out.println("Java sending...");
        vertx.eventBus().publish("news-feed", "Jammin' Java");
      }
    });
  }
}

File: ha/mods/com.burrsutter~senderjavascript~1.0/Sender.js
var eventBus = require("vertx/event_bus");
var console = require("vertx/console");
var vertx = require("vertx");

vertx.setPeriodic(1000, function sendMessage() {
	console.log("JavaScript sending...");
  eventBus.publish('news-feed', 'Jazzy JavaScript');
})
